docs/csi_ug_config_create_pvc.md

```diff
@@ -0,0 +1,112 @@
+# Creating a PersistentVolumeClaim (PVC)
+
+Create a PersistentVolumeClaim (PVC) yaml file for a persistent volume (PV).
+
+The IBM® block storage CSI driver supports using both file system and raw block volume types.
+
+**Important:** If not defined, the default type is `Filesystem`. Be sure to define the type as `Block` if this configuration is preferred.
+
+**Note:** The examples below create the PVC with a storage size 1 Gb. This can be changed, per customer needs.
+
+Use the sections below for creating yaml files for PVCs with file system and raw block volume types. After each yaml file creation, use the `kubectl apply` command.
+
+```
+kubectl apply -f <filename>.yaml
+```
+
+The `persistentvolumeclaim/<filename> created` message is emitted.
```

Comment:
[Same](https://github.com/IBM/ibm-block-csi-driver/pull/320/files#r648205475)
```suggestion
The `persistentvolumeclaim <filename> created` message is emitted.
```

Reply:
I think one conversation is enough for this topic